pyro/contrib/timeseries/gp.py

```diff
@@ -429,58 +441,3 @@ def forecast(self, targets, dts):
         filtering_state = self._filter(targets)
         predicted_mean, predicted_covar = self._forecast(dts, filtering_state)
         return MultivariateNormal(predicted_mean, predicted_covar)
-
-
-class LinearlyCoupledDependentMaternGP(DependentMaternGP):
```

Comment:
made this functionality available through `linearly_coupled=True` flat in `DependentMaternGP`